dnf-docker-test/features/steps/repo_steps.py

```diff
@@ -165,12 +186,15 @@ def given_repository_with_packages(ctx, repository):
     cmd = "{!s} {!s}".format(createrepo, tmpdir)
     step_i_successfully_run_command(ctx, cmd)
 
+    # set proper directory content ownership
+    file_utils.set_dir_content_ownership(ctx, tmpdir)
+
     repofile = REPO_TMPL.format(repository)
     ctx.table = Table(HEADINGS_INI)
     ctx.table.add_row([repository, "name",     repository])
     ctx.table.add_row(["",         "enabled",  "False"])
     ctx.table.add_row(["",         "gpgcheck", "False"])
-    ctx.table.add_row(["",         "baseurl",  "file://{!s}".format(tmpdir)])
+    ctx.table.add_row(["",         "baseurl",  "{!s}://{!s}".format(rtype, repopath)])
```

Comment:
I guess this will not work with `local`, since it will put `local://` instead of `file://`